### DISCUSSION TITLE
Separation of API into REST API and PORTAL

### DISCUSSION THREAD

**Discussion by IanMuller** [YetiForceCompany/YetiForceCRM#14906] [How to]
I have build some API connections with 6.1. Upgraded and things stops working.
Is there any guidance how to login from now on with REST?

**Comment by rskrzypczak** [comment]
https://doc.yetiforce.com/api/
https://github.com/YetiForceCompany/YetiForceCRM/discussions/14886#discussioncomment-961201

**Comment by IanMuller** [comment]
Ay... this is nice, never seen before. Tnx
